### DISCUSSION TITLE
10 version  update to 13 version

### DISCUSSION THREAD

**Discussion by cleartime** [vercel/next.js#47344] [Help]
### Summary

hi my app is 10 version  i want update to 13 version

please give me help

1. i update 11  to 12 then update 13
2. i update 11 to 12 then 13
3. i update 11 to 13
4. i reset my app in 13 version

**Comment by icyJoseph** [comment]
I think you can just upgrade to 13. 

The `next/image` and `next/link` have breaking changes that you'd need to fix, but everything else should work fine. 

The `next.config.js` plugin shapes have been standardized, so that might be problematic, but fixable by upgrading your plugins.

Last but not least, 13 requires React 18, and that might cause conflicts with dependencies, at there's a small chance, that you experience issues with the DOM you didn't know you had before.

**Reply by cleartime** [reply]
i try ，but not success, can you help me

**Reply by icyJoseph** [reply]
Sure, what's printing out :) - where have you gotten stuck?
